Fst2.h

```diff
@@ -251,6 +258,13 @@ void free_persistent_fst2(const char* filename);
 
 int get_graph_index(Fst2* fst2,int n_state);
 
+Fst2State new_Fst2State(Abstract_allocator prv_alloc);
+Fst2Tag new_Fst2Tag(Abstract_allocator prv_alloc);
+void set_initial_state(Fst2State e,int finality);
```

Comment:
Consider deleting the redundant prototype declarations of functions set_initial_state and set_final_state in Fst2.cpp

Reply:
done